BannerlordTwitch/BannerlordTwitch/Twitch/TwitchService.cs

```diff
@@ -370,6 +372,14 @@ public void SendReply(ReplyContext context, params string[] messages)
                     Log.Trace($"[{nameof(TwitchService)}] Chat: {Join(", ", messages)}");
                 }
             }
+
+            if (context.Source.RespondInExtension)
+            {
+                dynamic obj = new ExpandoObject();
+                obj.context = context;
+                obj.messages = messages;
+                BLTModule.bltApi.commandCallbackMessage = obj;
```

Comment:
Check my note above for why this is wrong. There needs to message passing in both directions, and this should be a separate message.

Reply:
I have to check how to do that. For now i put the return has todo.